main/webapp/modules/core/styles/index/open-project-ui.less

```diff
@@ -111,113 +111,168 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
- width: 80%
- }
+  width: 80%;
+}
 
 #metadata-body table > tbody > tr:nth-child(1) > th:nth-child(3) {
- width: 4%
- }
-
+  width: 4%;
+}
 
 /* Making metadata table scrollable*/
-.scrollable-table{
-	max-height: 70vh;
-	overflow-y: auto;
-	overflow-x: hidden;
+.scrollable-table {
+  max-height: 70vh;
+  overflow-y: auto;
+  overflow-x: hidden;
 }
 
 /* Making metadata table header sticky and darker*/
-#metadata-body table > tbody > tr >th{
-	position: -webkit-sticky;
-	position:sticky;
-	top:0;
-	color:black;
-	background:#b7b7b7;
+#metadata-body table > tbody > tr > th {
+  position: -webkit-sticky;
+  position: sticky;
+  top: 0;
+  color: black;
+  background: #b7b7b7;
 }
 
- 
- /* Project Tags */
+/* Project Tags */
 #projectTags {
-	position: relative;
-	padding: 3px 10px;
-	margin: 5px;
-	font-size: 130%;
-	border-radius: 5px;
-	-moz-border-radius: 5px;
-	-webkit-border-radius: 5px;
-	background: #bcf;
-	min-height: 18px;
-	height: auto;
-	overflow:auto;
+  position: relative;
+  padding: 3px 10px;
+  margin: 0px;
+  font-size: 130%;
+  border-radius: 0px;
+  -moz-border-radius: 0px;
+  -webkit-border-radius: 0px;
+  background: #bcf;
+  min-height: 18px;
+  height: auto;
+  overflow: auto;
 }
 
 ul#tagsUl {
-	margin: 0;
-	padding: 0;
+  padding: 0;
+  left: 60px; // add left to make room for magnifying glass icon
+  display: inline-block;
 }
 
 ul#tagsUl li {
-	list-style: none;
-	float: left;
-	text-align: center;
-	min-width: 80px;
+  list-style: none;
+  float: left;
+  text-align: center;
+  min-width: 80px;
 }
 
 ul#tagsUl li a {
-	padding: 6px 5px; 
-	display : inline-block;
-	text-align: center;
-	color: #444;
-	text-shadow: 0 -1px 2px rgba(255, 255, 255, 0.60);
-	cursor: pointer;
-	font-weight: bold;
-	min-width: 50px;
+  padding: 6px 5px;
+  display: inline-block;
+  text-align: center;
+  color: #444;
+  text-shadow: 0 -1px 2px rgba(255, 255, 255, 0.6);
+  cursor: pointer;
+  font-weight: bold;
+  min-width: 50px;
 }
 
-ul#tagsUl li a  {
-	text-decoration: none;
+ul#tagsUl li a {
+  text-decoration: none;
 }
 
 #projectTags .current {
-	background-color: #DEE6FF;
-	border-radius: 10px;
-	-moz-border-radius: 10px;
-	-webkit-border-radius: 10px;
+  background-color: #dee6ff;
+  border-radius: 10px;
+  -moz-border-radius: 10px;
+  -webkit-border-radius: 10px;
+}
+/* Search bar in open project */
+// search
+#divSearch {
+  float: left;
+  width: 2.5%;
+  margin-left: -0.5%;
+}
+#divImage {
+  float: left;
+  width: 90%;
+}
+
+#divInput {
+  float: right;
+  width: 10%;
+  display: none;
+}
+
+// input
+#searchInProjects {
+  border-radius: 10px;
+  margin: .5px auto 7px 14px;
+}
+
+.header-search-box {
+  display: none;
+  width: 0px;
+  float: left;
+}
+
+// icon search
+.magnifying_glass {
+  width: 90%;
+  height: 20px;
+  border-radius: 100%;
+  padding: 4px;
+}
+.magnifying_glass:hover {
+  cursor: pointer;
+}
+
+.magnifying-glass-open {
+  background-color: #5ca3cd;
```

Comment:
I would pick a more neutral color: this one sticks out as not being really part of the original color palette. Maybe just white (perhaps with transparency)?

Reply:
I used `#dee6ff` instead. The background color of the `current `search tag